src/librustc/lint/mod.rs

```diff
@@ -637,9 +637,22 @@ impl LintBuffer {
     }
 }
 
+fn level_to_flag(level: Level) -> &'static str {
```

Comment:
Would it be better for this be a method on `Level` itself?

(Perhaps a better initial question: Do we have other occurrences of this conversion in the code base?)

Reply:
I debated this; there are similar methods on `Level`. Since you thought of it I'll make it into a method.